pom.xml

```diff
@@ -2,6 +2,13 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.1.1</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+
```

Comment:
why this one instead of declaring <spring.boot-version> ?

Reply:
As SFlight is an educative example app independent of native-image we should then think about having this as a general approach (for the pom.xml generated by the Maven archetype).

Reply:
Yes we will need to discuss this.